purview/models/__init__.py

```diff
@@ -18,42 +18,53 @@
 story_flow = {
 
     'BugzillaBug': {
-        'forward_relationship': BugzillaBug.resolved_by_commits.definition['relation_type'],
+        'uid_name': BugzillaBug.id_.db_property or BugzillaBug.id.name,
```

Comment:
Optional: Getting the proper `uid_name` could be a separate function for less code repetition.

Reply:
All I have is a string while getting the uid_name. Having a separate function would then involve looping through all_models everytime in order to get the model instance.

Reply:
I meant something like
```python
def get_property_name(neomodel_property):
    return neomodel_property.db_property or neomodel_property.name

get_property_name(BugzillaBug.id_)
```